lib/rubocop/cop/rspec/capybara/visibility_matcher.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module Capybara
+        # Checks for boolean visibility in capybara finders.
+        #
+        # Capybara lets you find elements that match a certain visibiity using
+        # the `:visible` option. `:visible` accepts both boolean and symbols as
+        # values, however using booleans can have unwanted effects. `visible:
+        # false` does not find just invisible elements, but both visible and
+        # invisible elements. For expressiveness and clarity, use one of the
+        # symbol values, `:all`, `:hidden` or `:visible`.
+        # (https://www.rubydoc.info/gems/capybara/Capybara%2FNode%2FFinders:all)
+        #
+        # @example
+        #
+        #   # bad
+        #   expect(page).to have_selector('.foo', visible: false)
+        #
+        #   # bad
+        #   expect(page).to have_selector('.foo', visible: true)
+        #
+        #   # good
+        #   expect(page).to have_selector('.foo', visible: :all)
+        #
+        #   # good
+        #   expect(page).to have_selector('.foo', visible: :hidden)
+        #
+        #   # good
+        #   expect(page).to have_selector('.foo', visible: :visible)
+        #
+        class VisibilityMatcher < Cop
+          MSG = 'Use `:all`, `:hidden`, `:visible` instead of `true`, `false`.'
+
+          def_node_matcher :bad_method?, <<~PATTERN
```

Comment:
Should `bad_method?` be renamed? If so, to what?

Reply:
`visible_true?`/`visible_false?` should be sufficiently good.